tests/sentry/backup/test_models.py

```diff
@@ -52,6 +84,10 @@ def import_export_then_validate(self):
             if res.findings:
                 raise ValidationError(res)
 
+        # Return the actual JSON, so that we may use the `@targets_models` decorator to ensure that
+        # we have at least once instance of all the "tested for" models in the actual output.
```

Comment:
```suggestion
        # we have at least one instance of all the "tested for" models in the actual output.
```

Reply:
Feels like this should go in the docstring.

Reply:
Done.